lib/HLSL/DxilCondenseResources.cpp

```diff
@@ -2813,9 +3139,13 @@ class DxilCleanupAnnotateHandle : public ModulePass {
             if (isa<LoadInst>(Res))
               continue;
             CallInst *CRes = dyn_cast<CallInst>(Res);
+            if (!CRes)
+              continue;
             DxilInst_AnnotateHandle PrevAH(CRes);
             if (PrevAH) {
-              DXASSERT(AH.get_props() == PrevAH.get_props(), "otherwise, AnnotateHandle chain with inconsistent props");
+              DXASSERT(
+                  AH.get_props() == PrevAH.get_props(),
+                  "otherwise, AnnotateHandle chain with inconsistent props");
```

Comment:
Where does this happen now?  If we are still merging redundant annotate handles, shouldn't we be using tryMergeProps() here too?